certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3331 @@
+                },
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "id"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "type": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "type"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "description": "Retreive a single audio prompt object by providing an ID.",
+        "summary": "Get Audio Prompt By ID",
+        "operationId": "GetAudioPromptByID",
+        "parameters": [
+          {
+            "name": "prompt_id",
+            "in": "path",
+            "required": true,
+            "type": "integer",
+            "description": "Unique ID of the audio prompt.",
+            "x-ms-summary": "Audio prompt ID",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "x-ms-visibility": "important"
+      }
+    },
+    "/voip_endpoints/{endpoint_id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "version": {
+                  "type": "string",
+                  "description": "version"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "statusCode"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                },
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "id"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "employee_id": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "employee_id"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "description": "Retreive a single voip endpoint object by providing an ID.",

```

Comment:
@einarshamrihs please make sure to capitalize abbreviations if you use them to make everything clear. E.g. `VoIP` You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).